### DISCUSSION TITLE
podcast:transcript | podcastnamespace.org

### DISCUSSION THREAD

**Discussion by giscus** [jamescridland/podcastnamespace.org#6] [General]
# podcast:transcript | podcastnamespace.org

Adding new features to podcasting since 2020

https://podcastnamespace.org/tag/transcript?utm_source=podnews.net&utm_medium=email&utm_campaign=podnews.net%3A2024-01-26

**Comment by jrialan** [comment]
Hello, how is this supposed to work with DAI which can increase the length of the podcasts?

**Reply by jamescridland** [reply]
Great question. What Apple is doing is running some post-processing on it that matches the audio and transcript up to each other.
